hugr-py/src/hugr/hugr.py

```diff
@@ -131,6 +133,12 @@ def _get_typed_op(self, node: ToNode, cl: type[OpVar2]) -> OpVar2:
         assert isinstance(op, cl)
         return op
 
+    def nodes(self) -> Iterator[tuple[Node, NodeData]]:
```

Comment:
I think this is now indentical to `.items()` either remove or call `.items()` ?

Reply:
I think it's expected to have a `nodes` method.
I tried to use it from guppy, and mypy was suggesting `_nodes`...